lib/Core/Site/Values/Content.php

```diff
@@ -143,4 +176,42 @@ private function buildField(array $properties = [])
         $this->fields[$field->fieldDefIdentifier] = $field;
         $this->fieldsById[$field->id] = $field;
     }
+
+    private function getLocations()
```

Comment:
This method should have an optional `$limit` parameter I think (like `getChildren()`). In the current form it will limit to 25 locations, no? Well 25 is usually more than enought for 1 content, but sometimes not I think.